tests/meson.build

```
@@ -84,6 +84,17 @@ if gtest_dep.found()
 
     test('unittest_sink', unittest_sink, timeout: 120, env: testenv)
 
+    # RUN unittest_tensorRegion
+    if tflite2_support_is_available
+      unittest_tensorRegion = executable('unittest_tensorRegion',
```

Please use underscore_case instead of camelCase.
```suggestion
      unittest_tensor_region = executable('unittest_tensor_region',
```